From c19dd14ac90397fa7c55f701bec202b04ea5cf73 Mon Sep 17 00:00:00 2001
From: Sujeet Baranwal <sbaranwal@marvell.com>
Date: Tue, 4 May 2021 15:18:52 -0700
Subject: [PATCH 1518/1921] dts: marvell: Documentation for CM3 SRAM memory to
 cp115

CM3 SRAM address space would be used for Flow Control
configuration. Documentation is updated accordingly.

Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Signed-off-by: Sujeet Baranwal <sbaranwal@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/3419
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Tested-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Change-Id: Ia6d3b4bd9c1f5bbccf7daeb93ec475a71f711bcf
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/51110
Reviewed-by: Jia Ma <Jia.Ma@cavium.com>
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 Documentation/devicetree/bindings/net/marvell-pp2.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/marvell-pp2.txt b/Documentation/devicetree/bindings/net/marvell-pp2.txt
index b78397669320..9c76a13e4a02 100644
--- a/Documentation/devicetree/bindings/net/marvell-pp2.txt
+++ b/Documentation/devicetree/bindings/net/marvell-pp2.txt
@@ -37,6 +37,7 @@ Required properties (port):
   GOP (Group Of Ports) point of view. This ID is used to index the
   per-port registers in the second register area.
 - phy-mode: See ethernet.txt file in the same directory
+- cm3-mem: phandle to CM3 SRAM definitions
 
 Optional properties (port):
 
@@ -61,6 +62,7 @@ ethernet@f0000 {
 	      <0xc5000 0x100>;
 	clocks = <&gateclk 3>, <&gateclk 19>;
 	clock-names = "pp_clk", "gop_clk";
+	cm3-mem = <&cm3_sram>;
 
 	eth0: eth0@c4000 {
 		interrupts = <GIC_SPI 37 IRQ_TYPE_LEVEL_HIGH>;
-- 
2.31.1

